common/src/main/kotlin/entity/Permission.kt

```diff
@@ -150,4 +154,42 @@ sealed class Permission(val code: DiscordBitSet) {
     object ManageEmojis : Permission(0x40000000)
     object UseSlashCommands : Permission(0x80000000)
     object All : Permission(0xFFFFFDFF)
+
+    companion object {
+        val values: Set<Permission>
+            get() = setOf(
+                CreateInstantInvite,
+                KickMembers,
+                BanMembers,
+                Administrator,
+                ManageChannels,
+                ManageGuild,
+                AddReactions,
+                ViewAuditLog,
+                ViewChannel,
+                SendMessages,
+                SendTTSMessages,
+                ManageMessages,
+                EmbedLinks,
+                AttachFiles,
+                ReadMessageHistory,
+                MentionEveryone,
+                UseExternalEmojis,
+                ViewGuildInsights,
+                Connect,
+                Speak,
+                MuteMembers,
+                DeafenMembers,
+                MoveMembers,
+                UseVAD,
+                PrioritySpeaker,
+                ChangeNickname,
+                ManageNicknames,
+                ManageRoles,
+                ManageWebhooks,
+                ManageEmojis,
+                UseSlashCommands,
+                All,
```

Comment:
Should all be part of the set? It's technically not a Discord permission and we don't include it in intents.

Reply:
Good catch, I will edit it